src/main/java/uk/co/caprica/picam/CameraConfiguration.java

```diff
@@ -28,15 +28,14 @@
 import uk.co.caprica.picam.enums.Mirror;
 import uk.co.caprica.picam.enums.StereoscopicMode;
 
-import java.awt.geom.Rectangle2D;
-
 import static uk.co.caprica.picam.enums.Encoding.PNG;
 
 /**
  * Camera configuration.
  * <p>
  * This can be used in like a "builder".
  */
+@SuppressWarnings({"unused", "FieldCanBeLocal", "WeakerAccess"})
```

Comment:
I'm not sure Intellij specific inspections belong in a library. Also, there needs to be some justification/explanation for these - e.g. what access can be weaker here, what field can be local?